src/main/java/org/team1540/robot2022/commands/hood/HoodSetCommand.java

```diff
@@ -0,0 +1,22 @@
+package org.team1540.robot2022.commands.hood;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class HoodSetCommand extends CommandBase {
+    private final boolean state;
+    private final Hood hood;
+
+    public HoodSetCommand(Hood hood, Boolean state) {
```

Comment:
```suggestion
    public HoodSetCommand(Hood hood, boolean state) {
```
I believe `Boolean` is different than `boolean`